### PR TITLE
Fix(e2e): Fix playwright version

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
```release-note
NONE
```

### CODE REVIEW

**Comment by devinbinnie** [comment]
@yasserfaraazkhan Does this need to go in?

**Comment by yasserfaraazkhan** [comment]
Hi @devinbinnie 
Yes,
this wil fix few issues in the reporting. CMT is job is failing. I'll check it out separately.

**Review by toninis — APPROVED** [review]
❤️  Love it

File: e2e/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "dist/e2e_bundle.js",
   "scripts": {
     "clean": "rm -rf dist/ mochawesome-report/ node_modules/ testUserData/",
-    "run": "npm run build && npm run test",
+    "e2e:tests": "npm run build && npm run test",
```
Comment (devinbinnie) [inline e2e/package.json:8]: Naming seems weird

**Review by devinbinnie — APPROVED** [review]

File: e2e/package.json (hunk 1 above)
Comment (yasserfaraazkhan) [reply e2e/package.json]: renamed it to `run:e2e`

**Pull request merged**